modules/ROOT/pages/apid-behavioral-headers.adoc

```diff
@@ -1,32 +1,37 @@
-= Add Behavioral Headers to Simulated API Calls
+= Behavioral Headers for Simulated Calls to REST Methods
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
 :imagesdir: ../assets/images
 
-You can use the following behavioral headers to change the behavior of the mocking service. Use behavioral headers where you would normally specify other types of header when simulating requests to API specifications.
+You can use the following headers to change the behavior of the mocking service.
 
+== Introduce Delays into Response Times
```

Comment:
I think simply:

> Response Time Delays

Is easier to understand.

Reply:
Each of the headings in this topic tells the user what they can do. So, each heading starts with a verb.